src/components/Dashboard/TodoList.vue

```diff
@@ -39,7 +37,21 @@
           <div class="list-item">
             <div class="todo-label">
               <!-- title -->
-              <h3 class="list-item__title">{{ item.title }}</h3>
+              <h3 class="list-item__title">
+                {{ item.title }}
+
+                <!-- blue details button -->
+                <v-btn v-if="showDetailsBtnBlue(item)" class="expand-btn ml-1" text color="primary" :ripple=false>
+                  <v-icon>mdi-information-outline</v-icon>
+                  <span>{{ (panel === index) ? "Hide Details" : "View Details" }}</span>
+                </v-btn>
+
+                <!-- red details button -->
+                <v-btn v-if="showDetailsBtnRed(item)" class="expand-btn ml-1" text color="error" :ripple=false>
+                  <v-icon>mdi-information-outline</v-icon>
+                  <span>{{ (panel === index) ? "Hide Details" : "View Details" }}</span>
+                </v-btn>
```

Comment:
Looks like this:

![image](https://user-images.githubusercontent.com/10002889/195750104-bcaabb52-813e-4d0f-8452-483ab3844999.png)

![image](https://user-images.githubusercontent.com/10002889/195750118-23ce68a1-4d0c-455f-82a8-214661ab8f1d.png)

Instead of this:

![image](https://user-images.githubusercontent.com/10002889/195750155-f6fb9728-3939-491f-81cc-9c155b404736.png)

![image](https://user-images.githubusercontent.com/10002889/195750197-ea0f9451-da69-46f3-ae13-fbfc055aa559.png)

Expanded details:

![image](https://user-images.githubusercontent.com/10002889/195750260-4c074075-ce8a-4573-9671-3c2b05f48543.png)

![image](https://user-images.githubusercontent.com/10002889/195750290-7c949f87-6475-4fad-b732-73db9fb1a916.png)

This change was OKed by Yui.